tricolour/apps/tricolour/app.py

```diff
@@ -269,14 +272,10 @@ def _main(args):
                  "Interactive Python Debugger, as per user request")
         post_mortem_handler.disable_pdb_on_error()
 
-    log.info("Flagging on the {0:s} column".format(args.data_column))
-    if "/" in args.data_column:
-        data_string = args.data_column.split("/")
-        data_column = data_string[0]
-        string_columns = data_string[1]
-    else:
-        data_column = args.data_column
-        string_columns = None
+    # extract the name of the visibility column
+    # to support subtract_model_column
+    data_column = re.split(r'\+|-|/|\*|\(|\)', args.data_column)[0]
```

Comment:
This looks like a regex designed to detect the kind of expressions that https://github.com/ska-sa/dask-ms/blob/master/daskms/expressions.py#L103 supports. I think it's probably better to just split on the equals as follows as its easer to separate into the two cases

```python
parts = args.data_column.split("=")
log.info("Flagging on the {0:s} {0:s}".format(parts[0], "column" if len(parts) == 1 else "expression))
```

Additionally, I don't think the regex contained an equals sign so I think it would have failed on an assignment statement.

Could you please correct and test the two cases?

1. The simple case where data_column just references a column name
2. The complicated case where data_column contains an assignment.